Dockerfile

```diff
@@ -35,7 +35,7 @@ ENV DEBIAN_FRONTEND=noninteractive LANGUAGE=C.UTF-8 LANG=C.UTF-8 LC_ALL=C.UTF-8
     LC_CTYPE=C.UTF-8 LC_MESSAGES=C.UTF-8
 
 # By increasing this number we can do force build of all dependencies
-ARG DEPENDENCIES_EPOCH_NUMBER="2"
+ARG DEPENDENCIES_EPOCH_NUMBER="3"
```

Comment:
Those are wrong dependencies changed. PIP_DEPENDENCIES_EPOCH_NUMBER should be changed (which is way lower in the Docker file. That's causing much longer builds in Travis (until merged).

Reply:
Too much magic 🦄 Fixed!